deploy/assisted-service-configmap.yaml

```diff
@@ -12,4 +12,6 @@ data:
   OPENSHIFT_INSTALL_RELEASE_IMAGE: "quay.io/openshift-release-dev/ocp-release@sha256:eab93b4591699a5a4ff50ad3517892653f04fb840127895bb3609b3cc68f98f3"
   CREATE_S3_BUCKET: "true"
   ENABLE_AUTH: REPLACE_AUTH_ENABLED_FLAG
-  JWKS_URL: REPLACE_JWKS_URL # example https://example.com/.well-known/jwks.json
\ No newline at end of file
+  JWKS_URL: REPLACE_JWKS_URL # example https://example.com/.well-known/jwks.json
+  OCM_BASE_URL: "https://api-integration.6943.hive-integration.openshiftapps.com"
```

Comment:
this should have a `REPLACE_OCM_BASE_URL` replaced by `tools/deploy_assisted_installer_configmap.py`

Reply:
done